app/org/maproulette/framework/controller/TaskReviewController.scala

```diff
@@ -341,10 +341,13 @@ class TaskReviewController @Inject() (
         val csvRows = metrics.map { row =>
           displayedColumns.split(",").flatMap {
             case "Internal Id"   => Seq(row.review.taskId)
+            case "Feature Id"    => Seq(row.task.name)
             case "Review Status" => Seq(Task.reviewStatusMap(row.review.reviewStatus.get))
             case "Mapper"        => Seq(row.review.reviewRequestedByUsername.getOrElse(""))
             case "Challenge"     => Seq(row.review.challengeName.getOrElse(""))
+            case "Challenge Id"  => Seq(row.task.parent)
```

Comment:
In the PR description, would you add the new columns that were added to the csv? An example output (csv headers with one line of data) would be nice to see too.

Reply:
Partial screenshot was added, looks good